changelog/changes/2021-12-16-exclude-bridge-and-tunnels-from-DHCP.md

```diff
@@ -0,0 +1 @@
+- Excluded special network interface devices like bridge, tunnel, and veth devices from the default DHCP configuration to prevent networkd interference ([PR#56](https://github.com/flatcar-linux/init/pull/56))
```

Comment:
```suggestion
- Excluded special network interface devices like bridge, tunnel, vxlan, and veth devices from the default DHCP configuration to prevent networkd interference ([PR#56](https://github.com/flatcar-linux/init/pull/56))
```